encoding/x/protobuf/types.go

```diff
@@ -31,17 +31,20 @@ import (
 	"github.com/gogo/protobuf/proto"
 )
 
-// Encoding is the name of this encoding.
-const Encoding transport.Encoding = "protobuf"
+const (
+	// Encoding is the name of this encoding.
+	Encoding transport.Encoding = "protobuf"
 
-// GetApplicationError returns the application error from the server, if present.
+	rawResponseHeaderKey = "yarpc-protobuf-raw-response"
```

Comment:
This is a larger discussion we need to have before moving gRPC out of x/ as to how transports will understand application errors, as not all encodings are able to specify application errors in the same manner as thrift, leaving this for now.